build/test/builder/build_step_impl_test.dart

```diff
@@ -128,13 +128,15 @@ void main() {
 
         var aLib = await resolver.libraryFor(primary);
         expect(aLib.name, 'a');
-        expect(aLib.importedLibraries.length, 2);
-        expect(aLib.importedLibraries.any((library) => library.name == 'b'),
+        expect(aLib.definingCompilationUnit.libraryImports.length, 2);
+        expect(
+            aLib.definingCompilationUnit.libraryImports
+                .any((import) => import.library.name == 'b'),
```

Comment:
@scheglov This test is failing now, and if I do `print(aLib.definingCompilationUnit.libraryImports.map((l) => l.name).toList());` I get `[a, a]`. Any ideas?

Reply:
Nvm figured it out, I need to use `importedLibrary` not `library`.